src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,23 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  (comments.length > 0)
+    ? (
+      <div className="CommentList">
+        {comments.map(comment => (
+          <CommentInfo
+            comment={comment}
+            key={comment.id}
+          />
+        ))}
+      </div>
+    )
+    : (
+      <>
+        <hr />
+
+        <b data-cy="NoCommentsMessage">No comments yet</b>
```

Comment:
Avoid using `<b>` tag for bold text. It's better to use CSS for styling. You may add a class to this element and style it in your CSS.